ros_ws/src/car_control/launch/car_control.launch

```diff
@@ -1,42 +1,30 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <launch>
-<!-- 
+  <!-- 
 Launches all necessary nodes that listen for drive_parameters and translate them to the car
 -->
-    <arg name="mode_override" default="0"/>
-    <arg name="dms_check_rate" default="20" /> <!-- in Hz -->
-    <arg name="dms_expiration" default="100" /> <!-- in ms -->
-    <arg name="emergencystop_expiration" default="3000" /> <!-- in ms -->
-	
-    <!-- car control node -->
-    <node
-      respawn="true"
-      pkg="car_control"
-      type="car_controller"
-      name="car_controller"
-      output="screen" >
-    </node>
+  <arg name="mode_override" default="0"/>
+  <arg name="dms_check_rate" default="20" />
+  <!-- in Hz -->
+  <arg name="dms_expiration" default="100" />
+  <!-- in ms -->
+  <arg name="emergencystop_expiration" default="3000" />
+  <!-- in ms -->
 
-    <!-- dms control node -->
-    <node
-      respawn="true"
-      pkg="car_control"
-      type="dms_controller"
-      name="dms_controller"
-      output="screen" >
-      <param name="dms_check_rate" type="int" value = "$(arg dms_check_rate)" />
-      <param name="dms_expiration" type="int" value = "$(arg dms_expiration)" />
-      <param name="emergencystop_expiration" type="int" value = "$(arg emergencystop_expiration)" />
-      <param name="mode_override" type="int" value = "$(arg mode_override)" />
-    </node>
+  <!-- car control node -->
+  <node required="true" pkg="car_control" type="car_controller" name="car_controller" output="screen">
+  </node>
+
+  <!-- dms control node -->
+  <node required="true" pkg="car_control" type="dms_controller" name="dms_controller" output="screen">
+    <param name="dms_check_rate" type="int" value = "$(arg dms_check_rate)" />
+    <param name="dms_expiration" type="int" value = "$(arg dms_expiration)" />
+    <param name="emergencystop_expiration" type="int" value = "$(arg emergencystop_expiration)" />
+    <param name="mode_override" type="int" value = "$(arg mode_override)" />
+  </node>
+
+  <!-- Drive Parameters Multiplexer node -->
+  <node required="true" pkg="car_control" type="drive_parameters_multiplexer" name="drive_parameters_multiplexer" output="screen">
```

Comment:
Consider moving the `required` argument to the end of the tag so that the most important info shows up first.